Aptoide/aptoide-client-v8
# Feature/asv 1395 new home navigation

**What does this PR do?**

This PR aims to implement the new home view with two chips at the top to toggle games and apps views, along with the regular home;

**Database changed?**

No

**Where should the reviewer start?**

- [ ] HomeContainerFragment.java

**How should this be manually tested?**

Check that everything, from the scrolling to the toggle and events, is working as intended;

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-1395](https://aptoide.atlassian.net/browse/ASV-1395)

**Code Review Checklist**
